server/src/internalClusterTest/java/org/opensearch/cluster/metadata/ClusterIndexRefreshIntervalIT.java

```diff
@@ -69,9 +73,39 @@ public void setUp() throws Exception {
         internalCluster().startClusterManagerOnlyNode();
     }
 
+    static void putIndexTemplate(String refreshInterval) {
+        PutIndexTemplateRequest request = new PutIndexTemplateRequest("my-template"); // <1>
+        request.patterns(Arrays.asList("pattern-1", "log-*")); // <2>
+
+        request.settings(
+            Settings.builder() // <1>
+                .put("index.number_of_shards", 1)
+                .put("index.number_of_replicas", 1)
+                .put("index.refresh_interval", refreshInterval)
+        );
+        assertTrue(client().admin().indices().putTemplate(request).actionGet().isAcknowledged());
+    }
+
+    public void testIndexTemplateCreationWithLessThanMinimumRefreshInterval() throws ExecutionException, InterruptedException {
```

Comment:
Shouldn't the index template creation fail if the refresh interval is less than cluster minimum?

Reply:
it can succeed if there is a index template with lesser refresh interval

Reply:
there is not refresh interval set in the class, its -1. the class overriding this method will actually test the failure. i need to update the test name here to avoid confusion